doc/sample_systemd_script/urlmetadata.service

```diff
@@ -0,0 +1,22 @@
+# Sample service file for direct urlmetadata startup
```

Comment:
I think this file also should be templated and put in templates/system

Reply:
I think this file can be deleted, in fact. I'll go ahead and do that. Refer to here: 
https://gitlab.com/assembl/terraform/blob/dev/Infrastructure/Packer/UrlMetadata/scripts/urlmetadata_ubuntu_bootstrap.sh